contentcuration/contentcuration/static/js/edit_channel/uploader/views.js

```diff
@@ -380,13 +380,28 @@ var EditMetadataEditor = BaseViews.BaseView.extend({
         has_files = has_files || window.formatpresets.get({id:preset}).get("display");
       });
     }
+
```

Comment:
Determine if there are imported nodes: https://trello.com/c/60LZzI3r/192-when-importing-content-item-across-channels-some-metadata-should-be-locked-and-no-longer-be-editable-e-g-license-copyright-holde